.github/workflows/release-main.yaml

```diff
@@ -0,0 +1,22 @@
+name: release-main 
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-and-push-main:
+    runs-on: buildjet-4vcpu-ubuntu-2004
+    steps:
+      - uses: actions/checkout@v3
+      - uses: acorn-io/actions-setup@v2
+        with:
+          acorn-version: main
+      - uses: acorn-io/actions-login@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and Push
+        run: |
+          acorn build --push -t ghcr.io/${{ github.repository }}:main .
```

Comment:
sanity check: we don't need to worry about building for different architectures, do we?

Reply:
It depends - how are we going to be using this? Are just going to provision it ourselves and users will hit it or will users provision it for themselves.

Reply:
no. just worry about our own single arch usecase